libqtile/config.py

```diff
@@ -557,87 +557,95 @@ def __repr__(self):
 class Match:
     """Match for dynamic groups
 
-    It can match by title, class or role.
+    It can match by title, wm_class, role, wm_type, wm_instance_class or
+    net_wm_pid.
 
     ``Match`` supports both regular expression objects (i.e. the result of
-    ``re.compile()``) or strings (match as a "include" match). If a window
-    matches any of the things in any of the lists, it is considered a match.
+    ``re.compile()``) or strings (match as an "include"-match). If a window
+    matches all specified values, it is considered a match.
 
     Parameters
     ==========
     title:
-        things to match against the title (WM_NAME)
-    wm_class:
-        things to match against the second string in WM_CLASS atom
+        matches against the title (WM_NAME)
+    wmclass:
```

Comment:
Is there a reason to drop the underscore here?

Reply:
No this happened by accident, I'll revert that.